app/src/main/res/layout/fragment_user.xml

```diff
@@ -1,24 +1,200 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.core.widget.NestedScrollView xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
+
+    android:id="@+id/parent_scroll"
     android:layout_width="match_parent"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
-    android:background="@color/teal_200"
-    android:layout_height="match_parent"
-    tools:context=".UserFragment">
+    android:layout_height="match_parent">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        xmlns:app="http://schemas.android.com/apk/res-auto"
+        android:background="@color/white"
+        android:layout_height="match_parent"
+        tools:context=".UserFragment">
+
+
+        <LinearLayout
```

Comment:
if you're already using a ConstraintLayout above this, you don't need the LinearLayout and can achieve the same result by using the proper constraints, plus avoid nesting layouts.